Keas.Mvc/Controllers/TeamAdminController.cs

```diff
@@ -746,20 +752,340 @@ public async Task<IActionResult> Upload(IFormFile file)
+            var recAttributeAdded = false;
+            if (!string.IsNullOrWhiteSpace(r.Key1) && result.Success)
+            {
+                CreateAttribute(equipment, r.Key1, r.Value1, result, ref recAttributeCount, ref recAttributeAdded);
+            }
+            if (!string.IsNullOrWhiteSpace(r.Key2) && result.Success)
+            {
+                CreateAttribute(equipment, r.Key2, r.Value2, result, ref recAttributeCount, ref recAttributeAdded);
+            }
+            if (!string.IsNullOrWhiteSpace(r.Key3) && result.Success)
+            {
+                CreateAttribute(equipment, r.Key3, r.Value3, result, ref recAttributeCount, ref recAttributeAdded);
+            }
+            if (!string.IsNullOrWhiteSpace(r.Key4) && result.Success)
+            {
+                CreateAttribute(equipment, r.Key4, r.Value4, result, ref recAttributeCount, ref recAttributeAdded);
+            }
+            if (!string.IsNullOrWhiteSpace(r.Key5) && result.Success)
+            {
+                CreateAttribute(equipment, r.Key5, r.Value5, result, ref recAttributeCount, ref recAttributeAdded);
+            }
+            if (!string.IsNullOrWhiteSpace(r.Key6) && result.Success)
+            {
+                CreateAttribute(equipment, r.Key6, r.Value6, result, ref recAttributeCount, ref recAttributeAdded);
+            }
+
+            if (recAttributeAdded)
+            {
+                result.Messages.Add($"{recAttributeCount} Attribute(s) Added.");
+            }
+        }
+
+        private async Task<(int recPeopleCount, Person person)> GetCreatePerson(EquipmentImport r, Team team, EquipmentImportResults result)
+        {
+            Person person = null;
+            int recPeopleCount = 0;
+            if (!string.IsNullOrWhiteSpace(r.KerbUser))
+            {
+                try
+                {
+                    var personResult = await _identityService.GetOrCreatePersonFromKerberos(r.KerbUser, team.Id);
+                    recPeopleCount += personResult.peopleCount;
+                    person = personResult.Person;
+                }
+                catch (Exception)
+                {
+                    person = null;
+                    result.Success = false;
+                    result.ErrorMessage.Add($"!!!!!!!!!!!!!THERE IS A PROBLEM WITH KerbUser {r.KerbUser} PLEASE CONTACT PEAKS HELP with this User ID.!!!!!!!!!!!!!!!");
+                }
+
+                if (person == null)
+                {
+                    result.Success = false;
+                    result.ErrorMessage.Add($"KerbUser not found.");
+                }
+            }
+            else
+            {
+                result.Messages.Add("No Name");
+            }
+
+            return (recPeopleCount, person);
+
+        }
+
+        private async Task<int> AddEquipmentAssignment(EquipmentImport r, Person person, String userIdentity, String userName, Equipment equipment, EquipmentImportResults result)
+        {
```

Comment:
Need to check for either no kerb, or person is null and return
if (string.IsNullOrWhiteSpace(r.KerbUser))
            {
                return recAssignmentCount);
            }